scheduler/scheduler-server/src/test/resources/functionaltests/descriptors/Job_non_forked_script_task.xml

```diff
@@ -7,8 +7,8 @@
 		<task name="notforked">
 			<scriptExecutable>
 				<script>
-					<code language="groovy">
-						java.lang.System.exit(42);
+					<code language="scalaw">
```

Comment:
the problem here is that you change the semantic of the workflow, it would be better to create a separate workflow for scala